src/BannerViewManager.js

```diff
@@ -32,6 +32,7 @@ const getSizeForType = (type: AdType) => sizeForType[type] || sizeForType.standa
 type BannerViewProps = {
   type: AdType,
   placementId: string,
+  style: { [key: string]: any },
```

Comment:
@jakubsta why `number|string`? Do you have an example where `style` is defined as a string or number? I agree with `{[key: string]: number|string}` because styles are defined as an object but I am new to `react-native` so maybe I am missing something :)